resources/lang/de/admin.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'beatmapsets' => [
+        'covers' => [
+            'regenerate' => 'Regenerieren',
+            'regenerating' => 'Regeneriert...',
+            'remove' => 'Entfernen',
+            'removing' => 'Entfernt...',
+        ],
+        'show' => [
+            'covers' => 'Beatmapset-Banner bearbeiten',
+            'discussion' => [
+                '_' => 'Modding v2',
+                'activate' => 'aktivieren',
+                'activate_confirm' => 'modding v2 für diese beatmap aktivieren?',
+                'active' => 'aktiv',
+                'inactive' => 'inaktiv',
+            ],
+        ],
+    ],
+
+    'forum' => [
+        'forum-covers' => [
+            'index' => [
+                'delete' => 'Löschen',
+
+                'forum-name' => 'Forum #:id: :name',
+
+                'no-cover' => 'Kein Banner ausgewählt',
+
+                'submit' => [
+                    'save' => 'Speichern',
+                    'update' => 'Aktualisieren',
+                ],
+                
+                'title' => 'Liste der Forenbanner',
+
+                'type-title' => [
+                    'default-topic' => 'Standard-Forenbanner',
+                    'main' => 'Forenbanner',
+                ],
+            ],
+        ],
+    ],
+
+    'logs' => [
+        'index' => [
+            // todo: evtl. ein wort für "viewer" finden
+            'title' => 'Logs',
+        ],
+    ],
+
+    'pages' => [
+        'root' => [
+            'title' => 'Adminkonsole',
+
+            'sections' => [
+                'forum' => 'Forum',
+                'general' => 'Allgemein',
+                'store' => 'Shop',
+            ],
+        ],
+    ],
+
+    'store' => [
+        'orders' => [
+            'index' => [
+                'title' => 'Bestellungen',
+            ],
+        ],
+    ],
+
+    'users' => [
+        'restricted_banner' => [
+            // todo: übersetzung 'restricted'
```

Comment:
`Eingeschränkt` / `Gesperrt`?

Reply:
Gesperrt find ich super - das mach ich im nächsten commit dann

Reply:
Auf den zweiten Blick: Gesperrt passt doch nicht so gut. Erstens gibt es Doppeldeutigkeit mit gesperrten Threads, und zweitens passt es sinnlich nicht perfekt. In einen gesperrten Account kann man sich normalerweise nicht mal einloggen, hier ist er aber halt nur "restriced" - eingeschränkt.

Eingeschränkt verwende ich aus anderen Gründen nur ungerne - Sätze wie "eingeschränkte Benutzer können nicht antworten" will ich vermeiden